test/numbers.coffee

```diff
@@ -13,9 +13,26 @@
 # Binary Integer Literals
 # Binary notation is understood as would be decimal notation.
 
+toJS = (str) ->
+  CoffeeScript.compile str, bare: yes
+  .replace /^\s+|\s+$/g, '' # Trim leading/trailing whitespace
+
```

Comment:
Following the style of all other tests (except import/export), all these added tests should be removed.